### ISSUE TITLE
javascript.builtins.Date.toLocaleString - Chrome 88+ missing space for zh locales

### ISSUE THREAD

**Issue by ZnNozomi** [mdn/browser-compat-data#8882]
#### What information was incorrect, unhelpful, or incomplete?
The 'tolocaleString' method displays ' 2021/01/0411:03:32 'in Chrome version 88,and when I use the method 'new Date('2021/01/0411:03:32').getTime()' to get the timestamp, it returns NaN
#### What did you expect to see?
The 'tolocaleString' method is displayed in Chrome version 88 under '2021/01/04 11:03:32'
#### Did you test this? If so, how?
I tried this method on old and new versions, only version 88 below will return the correct date and time format, so is this a bug in the latest version of 'Chrome'?
<details>
<summary>MDN page report details</summary>

* Query: `javascript.builtins.Date.toLocaleString`
* MDN URL: https://developer.mozilla.org/en-us/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString
* Report started: 2021-01-28T01:45:15.927Z

</details>

**Comment by ddbeck** [comment]
Hi @ZnNozomi and thank you for reporting this. Unfortunately, I wasn't able to discover anything which suggested that the compat data was incorrect. For example, [inspired by the interactive example on MDN](https://developer.mozilla.org/en-us/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString), with these calls I got:

```
console.log((new Date('2021/01/04 11:03:32')).getTime());
// output: 1609758212000 
console.log((new Date('2021/01/04 11:03:32')).toLocaleString('en-US', { timeZone: 'UTC' }));
// output: 1/4/2021, 11:03:32 AM
```

And with your examples, verbatim (no space in the date string):

```
console.log((new Date('2021/01/0411:03:32')).getTime());
// output: NaN
console.log((new Date('2021/01/0411:03:32')).toLocaleString('en-US', { timeZone: 'UTC' }));
// output: Invalid Date
```

I got the same behaviors in different versions of Chrome and in Firefox. This suggests that things are working properly. That said, if you still think the compatibility data is incorrect, please follow-up with a test case which demonstrates the problem.

Thank you for helping us to confirm the correctness of the data!

**Comment by ZnNozomi** [comment]
@durbin Thank you very much for your reply. I'm sorry I just saw it.If you do that on version 88
console.log((new Date(1609758212000)).toLocaleString('zh-CN',{
        year: 'numeric',
        month: '2-digit',
        day: '2-digit',
        hour: '2-digit',
        minute: '2-digit',
        second: '2-digit',
        timezone:'UTC',
        hour12: false
      }));
// output 2021/01/0419:03:32 (will no space in the date string)

**Comment by ddbeck** [comment]
@ZnNozomi Thanks for the follow-up. I think I understand the problem now: the zh-CN output changed in Chrome 88 (compared to Chrome 87). The new output is wrong.

I found that [Chromium bug 1171086](https://bugs.chromium.org/p/chromium/issues/detail?id=1171086) reported this issue. It was merged into [bug 1170305](https://bugs.chromium.org/p/chromium/issues/detail?id=1170305); it appears that it was fixed a few days ago, but has not yet been released.

Here's what I would suggest as next steps:

1. You (or another contributor) can open a PR to add a note to `Date.json`'s [`toLocaleString` entry](https://github.com/mdn/browser-compat-data/blob/master/javascript/builtins/Date.json#L2557) for Chrome. The note might say something like, `From Chrome 88, some <code>zh</code> locales return the wrong date format. See <a href='crbug.com/1170305'>bug 1170305</a>.`
2. When the bug fix ships, the note can be amended. For example, "From Chrome 88 to X" to show that the problem is fixed.

**Comment by ZnNozomi** [comment]
@ddbeck Question found, feeling much better, thank you very much

**Comment by jculver050** [comment]
Close

**Comment by jusstes** [comment]
> @ZnNozomi Thanks for the follow-up. I think I understand the problem now: the zh-CN output changed in Chrome 88 (compared to Chrome 87). The new output is wrong.
> 
> I found that [Chromium bug 1171086](https://bugs.chromium.org/p/chromium/issues/detail?id=1171086) reported this issue. It was merged into [bug 1170305](https://bugs.chromium.org/p/chromium/issues/detail?id=1170305); it appears that it was fixed a few days ago, but has not yet been released.

Hi @ddbeck 
The current version of chrome (Chrome 102) no longer has this bug. It seems you can close the issue?

**Comment by majidzeno** [comment]
@ZnNozomi @ddbeck 
Testing this 
```
console.log((new Date(1609758212000)).toLocaleString('zh-CN',{
    year: 'numeric',
    month: '2-digit',
    day: '2-digit',
    hour: '2-digit',
    minute: '2-digit',
    second: '2-digit',
    timezone:'UTC',
    hour12: false
}));
```
returns `2021/01/04 13:03:32` I think this is the expected output.
Therefore, I suggest closing this issue. 🙏
